android/src/main/java/com/opentokreactnative/OTSubscriberViewManager.java

```diff
@@ -26,8 +26,9 @@ protected OTSubscriberLayout createViewInstance(ThemedReactContext reactContext)
 
     @ReactProp(name = "streamId")
     public void setStreamId(OTSubscriberLayout view, String streamId) {
-
+        if(streamId!=null){
```

Comment:
To keep formatting consistent we should add spaces around `!=`, so:

`if(streamId != null) {`

Reply:
done